persistence/sql/installer-workflow.md

```diff
@@ -16,34 +16,50 @@ WARNING: Read about [SQL Persistence NuGet Packages](/persistence/sql/#nuget-pac
 
 ## Contrasting Workflows
 
-Generally SQL installer script execution at endpoint startup will only be desired on a developers machine.
-
-So the code to enable this would be something similar to.
+In most cases, the automatic SQL installer scripts execution should be only enabled on developers machines. This kind of behavior can be enabled with the following code:
 
 snippet: InstallerWorkflow
 
-In this manner a developers machine can be identified by the existence of the `IsDevelopment` environment variable. A development machine could also be identified by checking a machine name convention, command line argument or any other toggle based on the specific environment.
+In this example the `IsDevelopment` environment variable is used to enable scripts execution on developers machines. A development machine could also be identified by checking a machine name convention, using some command line argument or incorporate another toggle mechanism that will be environment-specific.
+
+In higher environments (especially in production) the installation should be designed as an integral part of the deployment process. In particular, it is recommended to put in place mechanisms to verify generated scripts and test them before running in production. It may be also necessary to customize certain settings (e.g. schemas) or to modify the scripts themselves for every environment.
+
+Depending on the organization and the used tooling, the process can be fully automated, partially automated or even fully manual.
 
 
 ### Development Workflow
 
-A standard development work operates as follows:
+The main purpose in this type of environment is having the `Just hit F5` experience, by automatically creating required tables in a local database (or remote database used only for development). Keep in mind that the risk and potential impact of accidentally corrupting the database schema or existing data is minimal in this environment.
+
+A typical workflow on developers machine consists of the following steps:
+
+ 1. In the project add references to both `NServiceBus.Persistence.Sql` and `NServiceBus.Persistence.Sql.MsBuild` packages.
```

Comment:
Or maybe that should be below/above as a prerequisite, rather than one of the steps?